src/vs/workbench/contrib/chat/browser/media/chatSessions.css

```diff
@@ -173,6 +173,35 @@
 	animation: codicon-spin 1.5s steps(30) infinite;
 }
 
+/* Override global file icon hiding for chat session icons */
```

Comment:
@benibenj is there a way to do this directly through the provider? Basically ignoring the setting?